### DISCUSSION TITLE
显示被封禁，是什么原因呢？

### DISCUSSION THREAD

**Discussion by jiangxiaobo1994** [BurnHair/docs#114] [技术支持]
![image](https://github.com/BurnHair/docs/assets/105762008/cedc36d3-11ad-4264-b6df-3e41e4366a21)

**Comment by BennyThink** [comment]
写的很明确，你进行了 NSFW 的对话，并且你也有重复注册

**Reply by jiangxiaobo1994** [reply]
没有对话，应该是第三方API 调用辅助识别种子造成的。重复注册啥意思，我只通过github授权注册的。

**Reply by BennyThink** [reply]
重复注册就是一个人注册多个账号的意思。
